src/components/CommandBar/CommandBar.tsx

```diff
@@ -148,6 +150,7 @@ export class CommandBar extends React.Component<ICommandBarProps, ICommandBarSta
                          data-command-key={ index }
                          aria-haspopup={ !!(item.items && item.items.length) }
                          role='menuitem'
+                         aria-label={ `${ item.name }. ${ item.ariaLabel ? item.ariaLabel : '' }` }
```

Comment:
We should let the caller do this appending and determine if appropriate. They may want different. Can we just append the name and instructions into the ariaLabel?